gulp-ng-config.js

```diff
@@ -104,7 +104,7 @@ function gulpNgConfig (moduleName, configuration) {
     _.each(jsonObj, function (value, key) {
       constants.push({
         name: key,
-        value: JSON.stringify(value, null, spaces)
+        value: JSON.stringify(value, null, spaces).replace(/"/g, '\'')
```

Comment:
Totally - however it is a third party dependency for everyone - so we can't manipulate any inputs we don't control or it can lead to unexpected behavior.